source/standards/source-code/use-github.html.md.erb

```diff
@@ -6,17 +6,21 @@ review_in: 6 months
 
 # <%= current_page.data.title %>
 
-[GDS uses GitHub as its code repository](https://technology.blog.gov.uk/2016/05/31/how-we-use-git-at-the-government-digital-service/).
+GDS uses the [alphagov] organisation on [GitHub] to collaborate on code. The GOV.UK One Login programme uses [govuk-one-login].
 
-Put new repositories for GDS services in the [alphagov](https://github.com/alphagov/) organisation on GitHub.
+### Getting access to `alphagov`
 
-You can use your personal GitHub account to access alphagov but please ensure that you also link your GDS email address to your account. Ask your tech lead or technical architect to invite you. GDS will revoke your access to alphagov when you leave GDS.
+You can use your personal GitHub account to access alphagov if you wish.
 
-To secure your Github repository, make sure you:
+All github user accounts added to `alphagov` *must* be [connected](https://docs.github.com/en/account-and-profile/setting-up-and-managing-your-personal-account-on-github/managing-email-preferences/adding-an-email-address-to-your-github-account) with a valid `digital.cabinet-office.gov.uk` email address. Accounts not connected to a valid email address will be removed from `alphagov`.
```

Comment:
As above, is this requirement only for `alphagov`, or also for `govuk-one-login`?

(Separate point: it's GitHub, not github.)

Reply:
I'd ask @paroxp or someone for confirmation, but I believe we have the same rule, tho whether or not it's enforced I don't know.

It's unfortunate that the GitHub terraform thing we use attaches users via username rather than via email address, cos that would be a much more obvious initial control point I'd have thought.

(Or does it?? That's how we use it, but it looks like the [GitHub API does at least accept email addresses instead](https://docs.github.com/en/rest/orgs/members?apiVersion=2022-11-28#create-an-organization-invitation--parameters)?)

Reply:
I don't see how this requirement needs to differ. Having that requirement helps us track who does indeed need the access, and who should be rolled off.

To @galund's point - I believe we can setup invitations by email, and the users will be able to then decide how they want to log in. But I am pessimistic, if the email is then associated/recorded anywhere. Effectively meaning once people are invited, we lose sight of who they are.

> (Separate point: it's GitHub, not github.)

++

Reply:
Yeah I think doing everything (but especially initial invite) by email helps, because then you know at least at the start they'll have been set up right. And then _if_ we do offboarding correctly, well then we know that we've not got anyone in the org that shouldn't be.

(I still think the automated check is a good idea because people might get invited in various ways, but if we start the right way it seems more likely we won't have to fix people up later.)